botfront/imports/ui/components/stories/StoryEditorContainer.jsx

```diff
@@ -174,12 +174,16 @@ const StoryEditorContainer = ({
         });
     }, [branchPath]);
 
+    const getExceptionsForPath = path => exceptions[path] && ({
+        ...exceptions[path],
+        warnings: exceptions[path].warnings
+            .filter(storyMode === 'markdown' ? e => e : e => e.code !== 'no_such_response'), // don't show missing template warning in visual mode
+    });
```

Comment:
I feel like the ternary should be placed outside as it will always evaluate to the same for a said story
```
storyMode === 'markdown'  ? exceptions[path].warnings : exceptions[path].warnings .filter(e => e.code !== 'no_such_response')
```
```
storyMode === 'markdown' && exceptions[path].warnings || exceptions[path].warnings .filter(e => e.code !== 'no_such_response')

```

Reply:
This warning about missing responses is getting scrapped in another PR. I reverted the code to what it was, for now.